src/PrestaShopBundle/Form/Admin/Sell/Catalog/AttributeType.php

```diff
@@ -100,14 +106,17 @@ public function buildForm(FormBuilderInterface $builder, array $options)
                     . '&nbsp;' . $this->trans('Invalid characters:', 'Admin.Notifications.Info')
                     . ' ' . TypedRegexValidator::CATALOG_CHARS,
             ]);
-        if ($attributeGroup->group_type === AttributeGroupType::ATTRIBUTE_GROUP_TYPE_COLOR) {
-            $builder->add('color', ColorType::class, [
-                'label' => $this->trans('Color', 'Admin.Global'),
-                'required' => false,
-            ])->add('texture', FileType::class, [
-                'label' => $this->trans('Texture', 'Admin.Global'),
-                'required' => false,
-            ]);
+
+        if ($hasAttributeGroupId === true) {
```

Comment:
```suggestion
        if ($hasAttributeGroupId === true && $attributeGroup->group_type === AttributeGroupType::ATTRIBUTE_GROUP_TYPE_COLOR) {
```

Reply:
I'm not sure about the condition here, it makes sense to only add these fields when the parent is a color group BUT if we don't add it at all it means the fields are never present, it also means when you dynamically switch the attribute group in the page it can't toggle those fields

I'm not sure if the dynamic switch was handled in the page though but I guess it should since the legacy page handled this. So in this form type instead of removing the fields completely they should always be added, but their default visible state can be dynamized based on the attribute group type

Basically if the parent is not a group we could add a class `d-none` to hide the fields

Reply:
This bug is a known one (it's the 3rd bug here https://github.com/PrestaShop/PrestaShop/issues/35591) and will be fixed in an other PR.